src/sentry/api/serializers/models/organization.py

```diff
@@ -50,6 +50,7 @@ def serialize(self, obj, attrs, user):
             'name': obj.name or obj.slug,
             'dateCreated': obj.date_added,
             'isEarlyAdopter': bool(obj.flags.early_adopter),
+            'require2FA': bool(obj.flags.require_2fa),
```

Comment:
nit: do we use the key `require2FA` anywhere else? otherwise, I'd prefer a more consistent casing that treats acronyms as a word e.g. `require2fa`.

Reply:
ya, `require2FA` is consistent with the [organization details](https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/organization.py#L167) endpoint and some other places